pkg/formatted/color.go

```diff
@@ -16,19 +16,74 @@ package formatted
 
 import (
 	"io"
+	"sync"
 
 	"github.com/fatih/color"
 )
 
+type counter struct {
+	mu sync.RWMutex
+	n  int
+}
+
+func (c *counter) Increase() {
+	c.mu.Lock()
+	c.n = (c.n + 1) % len(allColors)
+	c.mu.Unlock()
+}
+
+func (c *counter) Get() int {
+	c.mu.RLock()
+	n := c.n
+	c.mu.RUnlock()
+	return n
+}
+
+var (
+	allColors = []color.Attribute{
```

Comment:
red is avoided since we keep it for error messages, (probably should have put this in the comment instead of here)